config/peerpods/podvm/README.md

```diff
@@ -34,3 +34,29 @@ Now when you create a KataConfig with `enablePeerPods: true` with empty
 `AZURE_IMAGE_ID` or `AWS_AMI_ID` in `peer-pods-cm`, then depending on the cloud
 provider configured, the operator will create the pod VM image based on the
 provided config.
+
+## PodVM image upload configuration
+
+The PodVM image can be embedded to a container image and used to unwrap and upload to the libvirt volume provided in the `peer-pods-secret`. 
+This feature is currently only supported for libvirt provider.
+
+To build an OCI image with PodVM image, `Dockerfile.podvm-oci` can be used as,
+```
+docker build -t podvm-libvirt \
+    --build-arg PODVM_IMAGE_SRC=<podvm_image_source> \
+    -f Dockerfile.podvm-oci .
+```
+Here the `PODVM_IMAGE_SRC` is where the `qcow2` image is present on the host and optionally `PODVM_IMAGE_PATH` can also be set which is the path of the qcow2 image inside the container which is to be then set as `<image_path>` as part of the `PODVM_IMAGE_URI` as below.
+
+
+Ensure that the `PODVM_IMAGE_URI` is set on the `libvirt-podvm-image-cm` in the format:
+```
+PODVM_IMAGE_URI: "<image_type>::<image_url>:<image_tag>::<image_path>"
+```
+eg:
+```
+PODVM_IMAGE_URI: "oci::quay.io/openshift_sandboxed_containers/libvirt-podvm-image:latest::/image/podvm-390x.qcow2"
```

Comment:
What made you include all these values? If you had just the standard image URL, the code can be simplified greatly. In other words, I don't see the value of image_type and image_path.

Reply:
To be clearer, the above is a comment to the design of the `PODVM_IMAGE_URI` variable, not just that line of text.

Reply:
Can we have a more standard URL? We could use the same format of Skopeo, and pass the complete URL directly to it (see the Skopeo README for docs). That would be `docker://`.

Reply:
@bpradipt requesting your views as well on this..

Reply:
@spotlesstofu I should have asked this one before my previous question, but here it is :-) 
Are you proposing `docker://`  or `oci::docker://...` ?

Reply:
Btw @ajayvic I'm missing the code that checks what's the image_type (eg. oci) and calls relevant function to handle the image_type.

Reply:
@bpradipt Just `docker://`, not `oci::docker://`.

This makes us consistent across projects (see [containers-transports.5](https://github.com/containers/image/blob/main/docs/containers-transports.5.md)).

For S3, we will use `s3://`, but that's not supported by skopeo.

Reply:
OK, let's keep it as it is.